lada/dike/stv/tally.py

```diff
@@ -58,12 +59,16 @@ def transfer_ballots(self, candidate, excess=0):
 
     def elect(self, candidate):
         self.elected.append(candidate)
+        self.log.info(f'Electing candidate {candidate} with score {candidate.score[-1][0]}')
+        self.results.update({candidate:candidate.score[-1][0]})
```

Comment:
For me, more informative would be one of the following statistics:

- for each candidate, for i in (1..) on how many ballots the candidate was on i-th place

- order of the candidates (ie. [elected 1, elected 2, elected 3, <blob of candidates with unknown order; or it can be even computed>, discarded 4, discarded 3, discarded 2, discarded 1])

- just the logs from the election

Reply:
I know, this is why I did not include this in the original code. It is hard to get an intuitive score with this method, due to transfers. Are the logs from the election stored somewhere?